src/main/java/com/ziminer/xork/world/Tile.java

```diff
@@ -39,4 +39,10 @@ public void addTransition(@Nonnull final TileTransition tileTransition) {
     public Collection<TileTransition> getAvailableTransitions() {
         return Collections.unmodifiableCollection(transitions.values());
     }
+
+    public String getKey (){
```

Comment:
Nit: insert space between () and {, and not between getKey and ().

Should be:
public String getKey() {